docs/sources/reference/components/otelcol/otelcol.receiver.solace.md

```diff
@@ -0,0 +1,210 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.solace/
+description: Learn about otelcol.receiver.solace
+title: otelcol.receiver.solace
+---
+
+# otelcol.receiver.solace
```

Comment:
Is it becoming stable right away?
I think you want to make it exprimental / public preview first?

Reply:
discussed internally